browser/ui/webui/settings/brave_default_extensions_handler.h

```diff
@@ -23,15 +25,25 @@ class BraveDefaultExtensionsHandler : public settings::SettingsPageUIHandler {
  private:
   // SettingsPageUIHandler overrides:
   void RegisterMessages() override;
-  void OnJavascriptAllowed() override;
-  void OnJavascriptDisallowed() override;
+  void OnJavascriptAllowed() override {}
+  void OnJavascriptDisallowed() override {}
 
   void GetRestartNeeded(const base::ListValue* args);
   void SetWebTorrentEnabled(const base::ListValue* args);
   void SetHangoutsEnabled(const base::ListValue* args);
   void SetIPFSCompanionEnabled(const base::ListValue* args);
   void SetMediaRouterEnabled(const base::ListValue* args);
   void SetBraveWalletEnabled(const base::ListValue* args);
+#if BUILDFLAG(ENABLE_TOR)
+  void SetTorEnabled(const base::ListValue* args);
+  void GetTorEnabled(const base::ListValue* args);
```

Comment:
nit - `IsTorEnabled` would be better here I think

Reply:
Fixed.